frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -173,6 +173,7 @@ export const detectComponents: FeatureDetector = async (dispatch) => {
         const noobaaSystems = await k8sList(NooBaaSystemModel, { ns: CEPH_STORAGE_NAMESPACE });
         if (noobaaSystems?.length > 0) {
           dispatch(setFlag(MCG_FLAG, true));
+          if (!cephFound) dispatch(setFlag(CEPH_FLAG, false));
```

Comment:
This flag should also be dispatched when clicking creation of a mcg standalone storage system/

Reply:
It will be set as `false` here only (for mcg standalone or any other deployment), as soon as noobaaSystem is found and no cephSystem is found. Why do we need to explicitly set it anywhere else ?

Reply:
added it in footer file, let me know if I should remove it from here or not ?